checker/src/org/checkerframework/checker/index/qual/LengthOf.java

```diff
@@ -0,0 +1,13 @@
+package org.checkerframework.checker.index.qual;
+
+/**
+ * An integer that, for each of the given sequences, is equal to the sequence's length. This is
```

Comment:
Put the second sentence in a paragraph of its own, and state that it is a temporary implementation detail that will be fixed in the future.